packages/hardhat-zksync-vyper/src/constanst.ts

```diff
@@ -0,0 +1,2 @@
+export const VYPER_EXTENSION = '.vy';
+export const SOLC_EXTENSION = '.sol';
```

Comment:
SOLIDITY_EXTENSION might be a better name...

Reply:
Agree